src/app.js

```diff
@@ -0,0 +1,2 @@
+const log = require('./log');
```

Comment:
możesz poczytać o modułach z ES2015 - http://2ality.com/2014/09/es6-modules-final.html i spróbować ich użyć ale to jako dodatkowe zadanie. Plus polecam artykuł - https://addyosmani.com/writing-modular-js/ jest dość obszerny i ma już kilka lat ale fajnie wprowadza w temat modularnego JS

Reply:
Dzięki za linki :) Spróbuję się zmierzyć z modułami ES2015 jak już ogarnę pozostałe zadania.